tests/configs/configs_test.go

```diff
@@ -85,4 +89,22 @@ var _ = Describe("Optional Configs", func() {
 			Expect(count).To(BeNumerically(">=", 1))
 		})
 	})
+	When("validating with kubeval", func() {
+		It("should pass", func() {
+			command := exec.Command("kubeval", "--strict", "--ignore-missing-schemas", "--skip-kinds", "Config", templatedPath)
```

Comment:
Let's use https://github.com/cloudfoundry/cf-for-k8s/blob/master/supported_k8s_versions.yml to test kubeval with both the newest and oldest versions of K8s that we support